DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/BasicConnectorBinding.cs

```diff
@@ -49,5 +53,100 @@ public BasicConnectorBinding(DocumentModelStore store, ArcGISSettings settings,
 
   public void RemoveModel(ModelCard model) => _store.RemoveModel(model);
 
-  public void HighlightModel(string modelCardId) => throw new System.NotImplementedException();
+  public async void HighlightModel(string modelCardId)
+  {
+    MapView mapView = MapView.Active;
+
+    var model = _store.GetModelById(modelCardId);
+
+    if (model is null)
+    {
+      return;
+    }
+
+    var objectIds = new List<string>();
+
+    if (model is SenderModelCard senderModelCard)
+    {
+      objectIds = senderModelCard.SendFilter.NotNull().GetObjectIds();
+    }
+
+    if (model is ReceiverModelCard receiverModelCard)
+    {
+      objectIds = receiverModelCard.ReceiveResult?.BakedObjectIds.NotNull();
+    }
+
+    if (objectIds is null)
+    {
+      return;
+    }
+
+    await QueuedTask
+      .Run(() =>
+      {
+        List<MapMember> mapMembers = GetMapMembers(objectIds, mapView);
+        ClearSelection();
+        SelectMapMembers(mapMembers);
+        mapView.ZoomToSelected();
+      })
+      .ConfigureAwait(false);
+  }
+
+  private List<MapMember> GetMapMembers(List<string> objectIds, MapView mapView)
+  {
+    List<MapMember> mapMembers = new();
+
+    foreach (string objectId in objectIds)
+    {
+      MapMember mapMember = mapView.Map.FindLayer(objectId);
+      if (mapMember is null)
+      {
+        mapMember = mapView.Map.FindStandaloneTable(objectId);
+      }
+      if (mapMember is null)
+      {
+        continue;
+      }
+      mapMembers.Add(mapMember);
+    }
+
+    return mapMembers;
+  }
+
+  private void ClearSelection()
+  {
+    List<Layer> mapMembers = MapView.Active.Map.GetLayersAsFlattenedList().ToList();
+    foreach (var member in mapMembers)
+    {
+      if (member is FeatureLayer featureLayer)
+      {
+        featureLayer.ClearSelection();
+      }
+    }
+  }
+
+  private void SelectMapMembers(List<MapMember> mapMembers)
+  {
+    foreach (var member in mapMembers)
+    {
+      if (member is FeatureLayer featureLayer)
+      {
+        using (RowCursor rowCursor = featureLayer.Search())
```

Comment:
Could we use the `inline` using syntax here?
i.e. 
```cs
using RowCursor rowCursor = featureLayer.Search();
```
It might help make this function easier to read

Reply:
agree, will fix it!